Index: app/src/main/java/com/webcalc/calculator/CalculatorController.java
===================================================================
--- app/src/main/java/com/webcalc/calculator/CalculatorController.java
+++ app/src/main/java/com/webcalc/calculator/CalculatorController.java
@@ -1,6 +1,7 @@
 package com.webcalc.calculator;

 import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
 import org.springframework.web.bind.annotation.RequestBody;
 import org.springframework.web.bind.annotation.RestController;

@@ -17,4 +18,9 @@
   public String calculate(@RequestBody String body) {
     return calculator.eval(body);
   }
+
+  @PutMapping("/maxFractionDigits")
+  public void setMaxFractionDigits(@RequestBody String body) {
+    calculator.setMaxFractionDigits(Integer.parseInt(body));
+  }
 }
