Index: app/src/test/java/com/webcalc/integration_tests/RestApiTest.java
===================================================================
--- app/src/test/java/com/webcalc/integration_tests/RestApiTest.java
+++ app/src/test/java/com/webcalc/integration_tests/RestApiTest.java
@@ -20,4 +20,21 @@
     .then()
         .body(equalTo("3"));
   }
+
+  @Test
+  void settingMaxFractionDigits() {
+    given()
+        .body("5")
+    .when()
+        .put("/maxFractionDigits")
+    .then()
+        .statusCode(200);
+
+    given()
+        .body("8 7 /")
+    .when()
+        .post("/eval")
+    .then()
+        .body(equalTo("1,14286"));
+  }
 }
