Index: app/src/main/java/com/webcalc/calculator/CalculatorController.java
===================================================================
--- app/src/main/java/com/webcalc/calculator/CalculatorController.java
+++ app/src/main/java/com/webcalc/calculator/CalculatorController.java
@@ -1,6 +1,7 @@
 package com.webcalc.calculator;

 import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
 import org.springframework.web.bind.annotation.RestController;

 @RestController
@@ -13,7 +14,7 @@
   }

   @PostMapping("/eval")
-  public String calculate() {
-    return calculator.eval("");
+  public String calculate(@RequestBody String body) {
+    return calculator.eval(body);
   }
 }
