Index: calculator/build.gradle
===================================================================
--- calculator/build.gradle
+++ calculator/build.gradle
@@ -3,6 +3,7 @@

 dependencies {
   testImplementation("org.junit.jupiter:junit-jupiter-api:${junitVersion}")
+  testImplementation("org.junit.jupiter:junit-jupiter-params:${junitVersion}")
   testRuntime("org.junit.jupiter:junit-jupiter-engine:${junitVersion}")
   testImplementation("org.assertj:assertj-core:${assertjVersion}")
 }
Index: calculator/src/test/java/com/webcalc/calculator/CalculatorShould.java
===================================================================
--- calculator/src/test/java/com/webcalc/calculator/CalculatorShould.java
+++ calculator/src/test/java/com/webcalc/calculator/CalculatorShould.java
@@ -1,15 +1,21 @@
 package com.webcalc.calculator;

-import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;

 import static org.assertj.core.api.Assertions.assertThat;

 class CalculatorShould {

-  @Test
-  void sum() {
+  @DisplayName("Sum")
+  @ParameterizedTest(name = "input: ''{0}'', expected result: ''{1}''")
+  @CsvSource({
+      "1 2 +, 3"
+  })
+  void sum(String input, String expectedResult) {
     Calculator calculator = new Calculator();
-    String result = calculator.eval("1 2 +");
-    assertThat(result).isEqualTo("3");
+    String result = calculator.eval(input);
+    assertThat(result).isEqualTo(expectedResult);
   }
 }
