Index: build.gradle
===================================================================
--- build.gradle
+++ build.gradle
@@ -28,6 +28,24 @@

 dependencies {
   compile('org.springframework.boot:spring-boot-starter-webflux')
-  testCompile('org.springframework.boot:spring-boot-starter-test')
-  testCompile('io.projectreactor:reactor-test')
+  testCompile('org.springframework.boot:spring-boot-starter-test') {
+    exclude group: "junit", module: "junit"
+  }
+  testCompile('org.junit.jupiter:junit-jupiter-api:5.2.0')
+  testRuntime('org.junit.jupiter:junit-jupiter-engine:5.2.0')
+  testCompile('io.projectreactor:reactor-test') {
+    exclude group: "junit", module: "junit"
+  }
+}
+
+test {
+  useJUnitPlatform()
+
+  testLogging {
+    events "passed", "skipped", "failed"
+  }
+
+  reports {
+    html.enabled = true
+  }
 }
