Index: calculator/src/main/java/com/webcalc/calculator/Calculator.java
===================================================================
--- calculator/src/main/java/com/webcalc/calculator/Calculator.java
+++ calculator/src/main/java/com/webcalc/calculator/Calculator.java
@@ -4,6 +4,7 @@
 import java.math.RoundingMode;
 import java.text.NumberFormat;
 import java.util.Locale;
+import java.util.function.BinaryOperator;

 public class Calculator {

@@ -20,23 +21,23 @@
     String[] tokens = input.split(" ");
     BigDecimal v1 = new BigDecimal(tokens[0]);
     BigDecimal v2 = new BigDecimal(tokens[1]);
-    BigDecimal result;
-    switch (tokens[2]) {
+    BinaryOperator<bigdecimal> f = function(tokens[2]);
+    BigDecimal result = f.apply(v1, v2);
+    return formatter.format(result);
+  }
+
+  private BinaryOperator<bigdecimal> function(String function) {
+    switch (function) {
       case "+":
-        result = v1.add(v2);
-        break;
+        return BigDecimal::add;
       case "-":
-        result = v1.subtract(v2);
-        break;
+        return BigDecimal::subtract;
       case "*":
-        result = v1.multiply(v2);
-        break;
+        return BigDecimal::multiply;
       case "/":
-        result = v1.divide(v2, maxFractionDigits, RoundingMode.HALF_UP);
-        break;
+        return (v1, v2) -> v1.divide(v2, maxFractionDigits, RoundingMode.HALF_UP);
       default:
-        throw new RuntimeException("Unsupported operation: " + tokens[2]);
+        throw new RuntimeException("Unsupported function: " + function);
     }
-    return formatter.format(result);
   }
 }
