Index: calculator/src/main/java/com/webcalc/calculator/Calculator.java
===================================================================
--- calculator/src/main/java/com/webcalc/calculator/Calculator.java
+++ calculator/src/main/java/com/webcalc/calculator/Calculator.java
@@ -13,6 +13,7 @@

   void setMaxFractionDigits(int maxFractionDigits) {
     this.maxFractionDigits = maxFractionDigits;
+    formatter.setMaximumFractionDigits(maxFractionDigits);
   }

   String eval(String input) {
