Index: calculator/src/main/java/com/webcalc/calculator/Calculator.java
===================================================================
--- calculator/src/main/java/com/webcalc/calculator/Calculator.java
+++ calculator/src/main/java/com/webcalc/calculator/Calculator.java
@@ -1,9 +1,13 @@
 package com.webcalc.calculator;

 import java.math.BigDecimal;
+import java.text.NumberFormat;
+import java.util.Locale;

 public class Calculator {

+  private NumberFormat formatter = NumberFormat.getNumberInstance(Locale.GERMANY);
+
   String eval(String input) {
     String[] tokens = input.split(" ");
     BigDecimal v1 = new BigDecimal(tokens[0]);
@@ -25,6 +29,6 @@
       default:
         throw new RuntimeException("Unsupported operation: " + tokens[2]);
     }
-    return result.toPlainString();
+    return formatter.format(result);
   }
 }
