Index: app/src/main/java/com/webcalc/calculator/CalculatorController.java
===================================================================
--- app/src/main/java/com/webcalc/calculator/CalculatorController.java
+++ app/src/main/java/com/webcalc/calculator/CalculatorController.java
@@ -5,9 +5,13 @@
 import org.springframework.web.bind.annotation.RequestBody;
 import org.springframework.web.bind.annotation.RestController;

+import javax.servlet.http.HttpSession;
+
 @RestController
 public class CalculatorController {

+  private static final String MAX_FRACTION_DIGITS = "maxFractionDigits";
+
   private final Calculator calculator;

   public CalculatorController(Calculator calculator) {
@@ -15,12 +19,12 @@
   }

   @PostMapping("/eval")
-  public String calculate(@RequestBody String body) {
-    return calculator.eval(body);
+  public String calculate(@RequestBody String body, HttpSession session) {
+    return calculator.eval(body, (Integer) session.getAttribute(MAX_FRACTION_DIGITS));
   }

   @PutMapping("/maxFractionDigits")
-  public void setMaxFractionDigits(@RequestBody String body) {
-    calculator.setMaxFractionDigits(Integer.parseInt(body));
+  public void setMaxFractionDigits(@RequestBody String body, HttpSession session) {
+    session.setAttribute(MAX_FRACTION_DIGITS, Integer.parseInt(body));
   }
