Index: src/main/java/com/webcalc/calculator/CalculatorController.java
===================================================================
--- src/main/java/com/webcalc/calculator/CalculatorController.java
+++ src/main/java/com/webcalc/calculator/CalculatorController.java
@@ -8,6 +8,6 @@

   @PostMapping("/eval")
   public String calculate() {
-    return "3";
+    return new Calculator().eval("");
   }
 }
