Index: calculator/src/test/java/com/webcalc/calculator/CalculatorShould.java
===================================================================
--- calculator/src/test/java/com/webcalc/calculator/CalculatorShould.java
+++ calculator/src/test/java/com/webcalc/calculator/CalculatorShould.java
@@ -11,7 +11,8 @@
   @DisplayName("Sum")
   @ParameterizedTest(name = "input: ''{0}'', expected result: ''{1}''")
   @CsvSource({
-      "1 2 +, 3"
+      "1 2 +, 3",
+      "2 3 +, 5",
   })
   void sum(String input, String expectedResult) {
     Calculator calculator = new Calculator();
