Index: calculator/src/main/java/com/webcalc/calculator/Calculator.java
===================================================================
--- calculator/src/main/java/com/webcalc/calculator/Calculator.java
+++ calculator/src/main/java/com/webcalc/calculator/Calculator.java
@@ -8,6 +8,12 @@
     String[] tokens = input.split(" ");
     BigDecimal v1 = new BigDecimal(tokens[0]);
     BigDecimal v2 = new BigDecimal(tokens[1]);
-    return v1.add(v2).toPlainString();
+    BigDecimal result;
+    if ("+".equals(tokens[2])) {
+      result = v1.add(v2);
+    } else {
+      result = v1.subtract(v2);
+    }
+    return result.toPlainString();
   }
 }
