Index: calculator/src/main/java/com/webcalc/calculator/Calculator.java
===================================================================
--- calculator/src/main/java/com/webcalc/calculator/Calculator.java
+++ calculator/src/main/java/com/webcalc/calculator/Calculator.java
@@ -8,6 +8,8 @@

 public class Calculator {

+  static final int DEFAULT_MAX_FRACTION_DIGITS = 2;
+
   private NumberFormat formatter = NumberFormat.getNumberInstance(Locale.GERMANY);

   private int maxFractionDigits;
Index: calculator/src/test/java/com/webcalc/calculator/CalculatorShould.java
===================================================================
--- calculator/src/test/java/com/webcalc/calculator/CalculatorShould.java
+++ calculator/src/test/java/com/webcalc/calculator/CalculatorShould.java
@@ -19,7 +19,7 @@
       "-13 122 +, 109",
   })
   void sum(String input, String expectedResult) {
-    String result = calculator.eval(input);
+    String result = calculator.eval(input, Calculator.DEFAULT_MAX_FRACTION_DIGITS);
     assertThat(result).isEqualTo(expectedResult);
   }

@@ -31,7 +31,7 @@
       "2 -2 -, 4",
   })
   void subtract(String input, String expectedResult) {
-    String result = calculator.eval(input);
+    String result = calculator.eval(input, Calculator.DEFAULT_MAX_FRACTION_DIGITS);
     assertThat(result).isEqualTo(expectedResult);
   }

@@ -44,7 +44,7 @@
       "-3 -5 *, 15",
   })
   void multiply(String input, String expectedResult) {
-    String result = calculator.eval(input);
+    String result = calculator.eval(input, Calculator.DEFAULT_MAX_FRACTION_DIGITS);
     assertThat(result).isEqualTo(expectedResult);
   }

@@ -59,8 +59,7 @@
       "3 2 /, 2, 0",
   })
   void divide(String input, String expectedResult, int maxFractionDigits) {
-    calculator.setMaxFractionDigits(maxFractionDigits);
-    String result = calculator.eval(input);
+    String result = calculator.eval(input, maxFractionDigits);
     assertThat(result).isEqualTo(expectedResult);
   }
 }
