Index: calculator/src/main/java/com/webcalc/calculator/Calculator.java
===================================================================
--- calculator/src/main/java/com/webcalc/calculator/Calculator.java
+++ calculator/src/main/java/com/webcalc/calculator/Calculator.java
@@ -12,23 +12,17 @@

   private NumberFormat formatter = NumberFormat.getNumberInstance(Locale.GERMANY);

-  private int maxFractionDigits;
-
-  void setMaxFractionDigits(int maxFractionDigits) {
-    this.maxFractionDigits = maxFractionDigits;
-    formatter.setMaximumFractionDigits(maxFractionDigits);
-  }
-
   String eval(String input, int maxFractionDigits) {
     String[] tokens = input.split(" ");
     BigDecimal v1 = new BigDecimal(tokens[0]);
     BigDecimal v2 = new BigDecimal(tokens[1]);
-    BinaryOperator<bigdecimal> f = function(tokens[2]);
+    BinaryOperator<bigdecimal> f = function(tokens[2], maxFractionDigits);
     BigDecimal result = f.apply(v1, v2);
+    formatter.setMaximumFractionDigits(maxFractionDigits);
     return formatter.format(result);
   }

-  private BinaryOperator<bigdecimal> function(String function) {
+  private BinaryOperator<bigdecimal> function(String function, int maxFractionDigits) {
     switch (function) {
       case "+":
         return BigDecimal::add;
