Index: app/src/main/java/com/webcalc/calculator/CalculatorController.java
===================================================================
--- app/src/main/java/com/webcalc/calculator/CalculatorController.java
+++ app/src/main/java/com/webcalc/calculator/CalculatorController.java
@@ -20,6 +20,8 @@

   @PostMapping("/eval")
   public String calculate(@RequestBody String body, HttpSession session) {
+    if (session.getAttribute(MAX_FRACTION_DIGITS) == null)
+      session.setAttribute(MAX_FRACTION_DIGITS, Calculator.DEFAULT_MAX_FRACTION_DIGITS);
     return calculator.eval(body, (Integer) session.getAttribute(MAX_FRACTION_DIGITS));
   }
