Index: app/build.gradle
===================================================================
--- app/build.gradle
+++ app/build.gradle
@@ -12,9 +12,9 @@
     exclude group: "junit", module: "junit"
   }
   testImplementation("io.rest-assured:rest-assured")
-  testImplementation("org.junit.jupiter:junit-jupiter-api:${junitVersion}")
-  testRuntime("org.junit.jupiter:junit-jupiter-engine:${junitVersion}")
-  testImplementation("org.assertj:assertj-core:${assertjVersion}")
+  testImplementation("org.junit.jupiter:junit-jupiter-api")
+  testRuntime("org.junit.jupiter:junit-jupiter-engine")
+  testImplementation("org.assertj:assertj-core")
 }

 test {
Index: build.gradle
===================================================================
--- build.gradle
+++ build.gradle
@@ -1,7 +1,7 @@
 buildscript {
   ext {
-    springBootVersion = "2.1.0.M2"
-    junitVersion = "5.3.0"
+    springBootVersion = "2.1.0.RELEASE"
+    junitVersion = "5.3.1"
     assertjVersion = "3.11.1"
   }
   repositories {
