Index: app/src/test/java/com/webcalc/integration_tests/RestApiTest.java
===================================================================
--- app/src/test/java/com/webcalc/integration_tests/RestApiTest.java
+++ app/src/test/java/com/webcalc/integration_tests/RestApiTest.java
@@ -24,7 +24,9 @@

   @Test
   void settingMaxFractionDigits() {
+    SessionFilter session = new SessionFilter();
     given()
+        .filter(session)
         .body("5")
     .when()
         .put("/maxFractionDigits")
@@ -32,6 +34,7 @@
         .statusCode(200);

     given()
+        .filter(session)
         .body("8 7 /")
     .when()
         .post("/eval")
